tests/e2e/tests/commands/new/new-routing.ts

```diff
@@ -0,0 +1,61 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import {ng} from '../../../utils/process';
+import {deleteDir} from '../../../utils/fs';
+
+export default function() {
+  const parentTestProjectDir = process.cwd();
+  const ngNewProjectName = 'new-test-project';
+
+  return Promise.resolve()
```

Comment:
You seem to overly complicate how this test could work. Why not:

```
return return Promise.resolve()
  .then(() => process.chdir(getGlobalVariable('tmp-root')))
  .then(() => ng('new', 'routing-project', '--routing'))
  .then(() => process.chdir(path.join('routing-project')))
  .then(() => ng('test', '--single-run'));
```

The only thing missing would be to `chdir` back to the directory we were in before this test, but really `e2e_runner` should take care of that (and I'm going to do a PR for it).

We already work off a temporary directory, there's no need to clean up after yourself :) This is by design.